### DISCUSSION TITLE
匿名评论报错

### DISCUSSION THREAD

**Discussion by ThanksGlory** [walinejs/waline#760] [Q&A] [answered]
Invalid value type for field 'nick',value '"\u533F\u540D"',expect type is {:type "Object"},but it is '{:type "String"}'. [400 POST https://glnoppa1.api.lncldglobal.com/1.1/classes/Comment]

**Comment by Mister-Hope** [comment, marked as answer]
nickname是一个字符串字段，从报错来看，leancloud 提示你的表里它是一个 Object 字段，建议你自查表内已有数据。

你可以先导出并移除旧数据再进行尝试，如果问题消失，请自行检查并调整旧数据的字段。
